packages/browser/src/integrations/dedupe.ts

```diff
@@ -23,6 +23,12 @@ export class Dedupe implements Integration {
    */
   public setupOnce(addGlobalEventProcessor: (callback: EventProcessor) => void, getCurrentHub: () => Hub): void {
     const eventProcessor: EventProcessor = currentEvent => {
+      // We want to ignore any non-error type events, e.g. transactions or replays
+      // These should never be deduped, and also not be compared against as _previousEvent.
+      if (currentEvent.type) {
```

Comment:
In these cases it's so weird that error events don't have a type....

Reply:
jup, backwards compatibility I guess 😅